src/main/resources/schemas/V64__referentie_tabel_systeem_column.sql

```diff
@@ -0,0 +1,29 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+-- add column is_systeem_waarde to referentie_tabel table to indicate if a referentie tabel should be considered a system value
+-- and should be treated as read-only in the application
+ALTER TABLE ${schema}.referentie_tabel
+ADD COLUMN is_systeem_tabel BOOL default false;
```

Comment:
Should we add a comment on this column with the text above?

Reply:
yes good one. we agreed that we wanted to add database comments. will do